pyproject.toml

```diff
@@ -16,6 +16,10 @@ maintainers = [{name = "NVIDIA Corporation & Affiliates"}]
 readme = "python/README.md"
 requires-python = ">=3.8"
 license = { file="LICENSE" }
+dependencies = [
+  'custatevec-cu11 ~= 1.4',
```

Comment:
Q: How about cutensornet? Am I right that cudaq is not yet using it?

Reply:
The tensornet backend is the only component using it, and right now, that code is not included in the build on this repo. We will have a dependency on it by default here after the next (bigger) cuQuantum update.